velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -480,9 +495,13 @@ std::shared_ptr<const core::PlanNode> SubstraitVeloxPlanConverter::toVeloxPlan(
       subfieldFilters.clear();
       remainingFilter =
           connectWithAnd(colNameList, veloxTypeList, scalarFunctions);
+      remainingFilter = connectWithAnd(
+          remainingFilter, colNameList, veloxTypeList, singularOrLists);
     } else {
       remainingFilter =
```

Comment:
This variable “remainingFilter” will not used.

Reply:
it used here https://github.com/oap-project/velox/pull/45/files#diff-6bbec4a6c456cb1efcf0c8f2dae7b80982e9002b9cf2a3460fd1ae545f291e49R512